eng/Subsets.props

```diff
@@ -352,7 +352,7 @@
       Test="true" Category="clr" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
     <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\ILCompiler.Compiler.Tests\ILCompiler.Compiler.Tests.csproj"
       Test="true" Category="clr" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(NativeAotSupported)' == 'true'"/>
-    <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\Mono.Linker.Tests\Mono.Linker.Tests.csproj"
+    <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\ILCompiler.Tests\ILCompiler.Tests.csproj"
```

Comment:
I would prefer different name. `ILCompiler.Tests` sounds like "This is the main test assembly for ILCompiler", which is not true. We could probably use `ILCompiler.Trim.Tests` as vast majority of the tests are around trimming, some are around AOT diagnostics, but that's a minority.